spec/models/category_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Category, :type => :model do
```

Comment:
should the name also be unique?

Reply:
def! updated validations and model test to check for uniqueness.